src/components/MDXComponents/__tests__/MDXCode.test.tsx

```diff
@@ -182,4 +182,19 @@ describe('MDXCode', () => {
     expect(highlightedLines[1]).toContainHTML('value');
     expect(highlightedLines[2]).toContainHTML('}');
   });
+
+  it('should throw an error for an empty codeblock string', async () => {
+    const mdxCode = <MDXCode codeString={''}></MDXCode>;
+    expect(() => {
+      render(mdxCode);
+    }).toThrow(emptyCodeString);
+  });
+
+  it('should throw an error if the codestring contains only whitespace', async () => {
+    const codeString = ' \t\n\r\v\f';
```

Comment:
What is this code string supposed to mean/represent?

Reply:
These are just whitespace characters, the idea is that we want to throw an error if a codeblock only contains whitespace.